resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/CircuitBreakerMetrics.java

```diff
@@ -54,7 +56,9 @@ private CircuitBreakerMetrics(String prefix, Iterable<CircuitBreaker> circuitBre
         circuitBreakers.forEach(circuitBreaker -> {
                 String name = circuitBreaker.getName();
                 CircuitBreaker.Metrics metrics = circuitBreaker.getMetrics();
-
+                //state as an integer
+                metricRegistry.register(name(prefix, name, STATE),
```

Comment:
Hi @eleduardo,
I'd prefer to add explicit `final int` field to `CircuitBreaker.State` so can be sure that enum elements reordering won't affect any of our users.